WordPress/Classes/ViewRelated/Reader/ReaderCommentCell.swift

```diff
@@ -0,0 +1,217 @@
+import UIKit
+import WordPressShared
+import Gridicons
+
+@objc protocol ReaderCommentCellDelegate: WPRichContentViewDelegate
+{
+    func cell(cell: ReaderCommentCell, didTapAuthor comment: Comment)
+    func cell(cell: ReaderCommentCell, didTapLike comment: Comment)
+    func cell(cell: ReaderCommentCell, didTapReply comment: Comment)
+}
+
+class ReaderCommentCell : UITableViewCell
+{
+    var enableLoggedInFeatures = false
+
+    @IBOutlet var avatarImageView: UIImageView!
+    @IBOutlet var authorButton: UIButton!
+    @IBOutlet var timeLabel: UILabel!
+    @IBOutlet var textView: WPRichContentView!
+    @IBOutlet var replyButton: UIButton!
+    @IBOutlet var likeButton: UIButton!
+    @IBOutlet var actionBar: UIStackView!
+    @IBOutlet var leadingContentConstraint: NSLayoutConstraint!
+
+    weak var delegate: ReaderCommentCellDelegate? {
+        didSet {
+            textView.delegate = delegate
+        }
+    }
+
+    var comment: Comment?
+
+    var showReply: Bool {
+        get {
+            if let comment = comment, let post = comment.post as? ReaderPost {
+                return post.commentsOpen && enableLoggedInFeatures
+            }
+            return false
+        }
+    }
+
+    override var indentationLevel: Int {
+        didSet {
+            leadingContentConstraint.constant = CGFloat(indentationLevel) * indentationWidth
+        }
+    }
+
+    override var indentationWidth: CGFloat {
+        didSet {
+            leadingContentConstraint.constant = CGFloat(indentationLevel) * indentationWidth
```

Comment:
Hmm, maybe some way to not duplicate these two identical `leadingContentConstraint.constant` code blocks. Not sure of a clean way though.